faculty/clients/experiment.py

```diff
@@ -94,6 +100,40 @@ def make_experiment_run(self, data):
         return ExperimentRun(**data)
 
 
+class ExperimentRunTagSchema(BaseSchema):
```

Comment:
nit: Can we reorder these the same as the classes above?

Reply:
Done